functions

```diff
@@ -29,10 +29,7 @@ emergency_shell() {
 
 detect_virt() {
    # Detect LXC containers
-   [ ! -e /proc/self/environ ] && return
```

Comment:
I am not sure about removing this line.
Can we be sure that we are not at a point during boot that has not yet mounted */proc*?

Reply:
I am not using /proc so it is irrelevant ...